src/ICeWall/doc/init_footprint.md

```diff
@@ -0,0 +1,11 @@
+## ICeWall init_footprint
+### Synopsys
+```
+  % ICeWall init_footprint [<signal_mapping_file>]
```

Comment:
Is having both angle and square brackets proper notation?

Reply:
If the argument were required I would say:

ICeWall init_footprint <signal_mapping_file>

I'm happy to comply with whatever standard we want to use here.

Reply:
This notation makes sense to me.

Reply:
I thought optional arguments were square brackets only
`[optional_argument]`

Reply:
I would think that is an optional literal